api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -154,26 +153,28 @@ def aspirate(
         rate: float = 1.0,
     ) -> InstrumentContext:
         """
-        Aspirate a given volume of liquid from the specified location, using
-        this pipette.
+        Draw liquid into a pipette tip. Includes parameters that control
+        aspiration volume, well location, tip position in a well,and pipette flow rate.
+        See :ref:`new-aspirate`.
 
-        :param volume: The volume to aspirate, in microliters (µL).  If 0 or
-                       unspecified, defaults to the highest volume possible
-                       with this pipette and its currently attached tip.
+        :param volume: The volume to aspirate, measured in µL. If 0 or
+                       unspecified, defaults to the maximum volume for
+                       the pipette and its currently attached tip.
         :type volume: int or float
-        :param location: Where to aspirate from. If `location` is a
-                         :py:class:`.Well`, the robot will aspirate from
-                         :py:obj:`well_bottom_clearance.aspirate <well_bottom_clearance>`
-                         mm above the bottom of the well. If `location` is a
-                         :py:class:`.Location` (i.e. the result of
-                         :py:meth:`.Well.top` or :py:meth:`.Well.bottom`), the
-                         robot will aspirate from the exact specified location.
-                         If unspecified, the robot will aspirate from the
-                         current position.
-        :param rate: A relative modifier for how quickly to aspirate liquid.
-                     The flow rate for this aspirate will be
-                     `rate` * :py:attr:`flow_rate.aspirate <flow_rate>`.
-                     If not specified, defaults to 1.0.
+        :param location: Tells the robot where to aspirate from. The location can be a
+                         :py:class:`.Well` or a :py:class:`.Location`.
+
+                            - If the location is a ``well``, the robot will aspirate at or above
+                              the center of the bottom of the well. The distance (in mm) from the
+                              well bottom is specified by the :py:obj:`well_bottom_clearance.aspirate` object.
+
+                            - If the location is a ``location`` (e.g., the result of :py:meth:`.Well.top`
+                              or :py:meth:`.Well.bottom`), the robot will aspirate from that specified position.
```

Comment:
the classes `Well` and `Location` should be capitalized. it wouldn't matter overly much, except here lowercase `location` is the name of the _parameter_ in question. (naming things is hard.)
```suggestion
                            - If the location is a ``Well``, the robot will aspirate at or above
                              the bottom center of the well. The distance (in mm) from the
                              well bottom is specified by :py:obj:`well_bottom_clearance.aspirate`.

                            - If the location is a ``Location`` (e.g., the result of :py:meth:`.Well.top`
                              or :py:meth:`.Well.bottom`), the robot will aspirate from that specified position.
```
🐍 🤓 `well_bottom_clearance.aspirate` isn't an object, it's a parameter of an object.